cmd/preferredimports/preferredimports.go

```diff
@@ -0,0 +1,251 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// verify that all the imports have our preferred alias(es).
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"go/ast"
+	"go/build"
+	"go/format"
+	"go/parser"
+	"go/token"
+	"golang.org/x/crypto/ssh/terminal"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+)
+
+var (
+	importAliases = flag.String("import-aliases", "hack/.import-aliases", "json file with import aliases")
+	confirm       = flag.Bool("confirm", false, "update file with the preferred aliases for imports")
+	regex         = flag.String("include-path", "(test/e2e|test/e2e_node)", "only files with paths matching this regex is touched")
```

Comment:
I wouldn't touch test/e2e_kubeadm just yet
```suggestion
	regex         = flag.String("include-path", "(test/e2e/|test/e2e_node)", "only files with paths matching this regex is touched")
```